janitor/functions.py

```diff
@@ -1617,8 +1620,8 @@ def limit_column_characters(
 
 @pf.register_dataframe_method
 def row_to_names(
-    df: pd.DataFrame,
-    row_number: int = None,
+    df,
```

Comment:
Not sure why type is removed here either.

Reply:
That must have been my fault.

```suggestion
    df: pd.DataFrame,
```